### ISSUE TITLE
Some Esri Cedar Examples Broken

### ISSUE THREAD

**Issue by MKellyEsri** [Esri/cedar#135]
The 'Filter Chart by BBOX' example at <a href="http://esri.github.io/cedar/examples/filter-chart-by-bbox.html">the following link</a> is not working as expected. Please see below for errors in Chrome Developer Tools (in case it fixes itself!).

![image](https://cloud.githubusercontent.com/assets/9192183/8648714/db504146-295a-11e5-8632-be17b2423912.png)

I happened to come across this as I have been doing some customisation of this template and couldn't figure out what had changed since last Friday! I reverted back to a version that was definitely working and continued to get the same error.

I have gone through some of the samples on GitHub and similar errors are being returned for the following examples:

Map Interaction
Filter by Geometry
Filter by Extent
Map and Multiple Charts

**Comment by ajturner** [comment]
Thanks! I'll look into it - though today we'll likely merge in Vega2 upgrade.

We also need to start adding unit and integration tests :)

**Comment by MKellyEsri** [comment]
Great thanks for the quick reply Andrew :-)

**Comment by ajturner** [comment]
@MKellyEsri we just pushed Cedar 0.2 and these should be fixed now - can you confirm?

**Comment by MKellyEsri** [comment]
Thanks for getting back to me Andrew. That seems to have done the trick - I just had to update the cedar.js file and change http://cdnjs.cloudflare.com/ajax/libs/vega/1.4.3 to http://vega.github.io/vega/.

I also added in:

```javascript
if (d === undefined || d === null) {
	chart.clearSelection();
	return;
}
```

into the chart.on('mouseover') function as in the example as d returns null for any mouseover events unless it is within a specific bar.

However, there seems to be some handling problems when hovering over the map and chart in [this](http://esri.github.io/cedar/examples/map-to-chart-interaction.html) example (I am using something similar). Check out [this video](http://screencast.com/t/upShZqinbi).
